packages/neuron-wallet/src/mock.ts

```diff
@@ -8,25 +8,30 @@ export const transactions = Array.from({
 
 export const transactionCount = 500
 
-interface Wallet {
+export interface MockWallet {
```

Comment:
While we're leaving many wallet implementation detail for later, we should just call this `Wallet` instead of `MockWallet`.